lib/tests/local_request_private_cookie-issue-368.rs

```diff
@@ -0,0 +1,30 @@
+#![feature(plugin, custom_derive, decl_macro)]
+#![plugin(rocket_codegen)]
+
+extern crate rocket;
+
+use rocket::http::{Cookie, Cookies};

```

Comment:
Move the `Cookie` input into the `tests` module below since it's not used until then.